devices/tuya.js

```diff
@@ -2247,4 +2247,22 @@ module.exports = [
         fromZigbee: [fz.tuya_remote],
         toZigbee: [],
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_whkgqxse'}],
+        model: 'Zigbee',
+        vendor: 'ELIVCO',
+        description: 'Smart Temperature&Humidity Sensor',
+        fromZigbee: [
+            fz.elivco_lcd_temperature_humidity_sensor,
```

Comment:
Can't you use `fz.nous_lcd_temperature_humidity_sensor` instead?

Reply:
I tried it, but I was only getting "null" instead of the values

Reply:
Could you provide the debug log of this ?

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable debug logging.

Reply:
debug 2022-04-28 18:21:28: Received Zigbee message from 'Bedroom Thermometer', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,239],"type":"Buffer"},"datatype":2,"dp":1},{"data":{"data":[0,0,0,53],"type":"Buffer"},"datatype":2,"dp":2},{"data":{"data":[0,0,0,100],"type":"Buffer"},"datatype":2,"dp":4},{"data":{"data":[0,0,1,104],"type":"Buffer"},"datatype":2,"dp":10},{"data":{"data":[0,0,0,200],"type":"Buffer"},"datatype":2,"dp":11}],"seq":15106}' from endpoint 1 with groupID null
info  2022-04-28 18:21:28: MQTT publish: topic 'zigbee2mqtt/Bedroom Thermometer', payload '{"battery":100,"humidity":53,"last_seen":"2022-04-28T18:21:28+02:00","linkquality":255,"temperature":"23.5","temperature_unit_convert":"celsius"}'
debug 2022-04-28 18:21:28: Received Zigbee message from 'Bedroom Thermometer', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,70],"type":"Buffer"},"datatype":2,"dp":12},{"data":{"data":[0,0,0,20],"type":"Buffer"},"datatype":2,"dp":13},{"data":{"data":[0,0,0,30],"type":"Buffer"},"datatype":2,"dp":17},{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":15362}' from endpoint 1 with groupID null
info  2022-04-28 18:21:28: MQTT publish: topic 'zigbee2mqtt/Bedroom Thermometer', payload '{"battery":100,"humidity":53,"last_seen":"2022-04-28T18:21:28+02:00","linkquality":255,"temperature":"23.5","temperature_unit_convert":"celsius"}'
debug 2022-04-28 18:21:29: Received Zigbee message from 'Bedroom Thermometer', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,239],"type":"Buffer"},"datatype":2,"dp":1},{"data":{"data":[0,0,0,53],"type":"Buffer"},"datatype":2,"dp":2},{"data":{"data":[0,0,0,100],"type":"Buffer"},"datatype":2,"dp":4},{"data":{"data":[0,0,1,104],"type":"Buffer"},"datatype":2,"dp":10},{"data":{"data":[0,0,0,200],"type":"Buffer"},"datatype":2,"dp":11}],"seq":15618}' from endpoint 1 with groupID null
info  2022-04-28 18:21:29: MQTT publish: topic 'zigbee2mqtt/Bedroom Thermometer', payload '{"battery":100,"humidity":53,"last_seen":"2022-04-28T18:21:29+02:00","linkquality":255,"temperature":"23.5","temperature_unit_convert":"celsius"}'
debug 2022-04-28 18:21:29: Received Zigbee message from 'Bedroom Thermometer', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,70],"type":"Buffer"},"datatype":2,"dp":12},{"data":{"data":[0,0,0,20],"type":"Buffer"},"datatype":2,"dp":13},{"data":{"data":[0,0,0,30],"type":"Buffer"},"datatype":2,"dp":17},{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":15874}' from endpoint 1 with groupID null
info  2022-04-28 18:21:29: MQTT publish: topic 'zigbee2mqtt/Bedroom Thermometer', payload '{"battery":100,"humidity":53,"last_seen":"2022-04-28T18:21:29+02:00","linkquality":255,"temperature":"23.5","temperature_unit_convert":"celsius"}'
debug 2022-04-28 18:21:30: Received Zigbee message from 'Bedroom Thermometer', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,238],"type":"Buffer"},"datatype":2,"dp":1},{"data":{"data":[0,0,0,53],"type":"Buffer"},"datatype":2,"dp":2},{"data":{"data":[0,0,0,100],"type":"Buffer"},"datatype":2,"dp":4},{"data":{"data":[0,0,1,104],"type":"Buffer"},"datatype":2,"dp":10},{"data":{"data":[0,0,0,200],"type":"Buffer"},"datatype":2,"dp":11}],"seq":16130}' from endpoint 1 with groupID null
info  2022-04-28 18:21:30: MQTT publish: topic 'zigbee2mqtt/Bedroom Thermometer', payload '{"battery":100,"humidity":53,"last_seen":"2022-04-28T18:21:30+02:00","linkquality":255,"temperature":"23.5","temperature_unit_convert":"celsius"}'
debug 2022-04-28 18:21:30: Received Zigbee message from 'Bedroom Thermometer', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,70],"type":"Buffer"},"datatype":2,"dp":12},{"data":{"data":[0,0,0,20],"type":"Buffer"},"datatype":2,"dp":13},{"data":{"data":[0,0,0,30],"type":"Buffer"},"datatype":2,"dp":17},{"data":{"data":[0],"type":"Buffer"},"datatype
Is this correct?

Reply:
I don't see any null values here?

Reply:
Sorry, in the previous log it just kept sending the last know values it received before I changed the fz.
I removed and re-added the device, now you can see the null values:
`Zigbee2MQTT:debug 2022-04-29 22:50:13: Received Zigbee message from '0xa4c138bb84e2f10d', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,229],"type":"Buffer"},"datatype":2,"dp":1},{"data":{"data":[0,0,0,53],"type":"Buffer"},"datatype":2,"dp":2},{"data":{"data":[0,0,0,100],"type":"Buffer"},"datatype":2,"dp":4},{"data":{"data":[0,0,1,104],"type":"Buffer"},"datatype":2,"dp":10},{"data":{"data":[0,0,0,200],"type":"Buffer"},"datatype":2,"dp":11}],"seq":59910}' from endpoint 1 with groupID null
Zigbee2MQTT:info  2022-04-29 22:50:13: MQTT publish: topic 'zigbee2mqtt/0xa4c138bb84e2f10d', payload '{"battery":null,"humidity":null,"last_seen":"2022-04-29T22:50:13+02:00","linkquality":255,"temperature":null}'
Zigbee2MQTT:debug 2022-04-29 22:50:13: Received Zigbee message from '0xa4c138bb84e2f10d', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,70],"type":"Buffer"},"datatype":2,"dp":12},{"data":{"data":[0,0,0,20],"type":"Buffer"},"datatype":2,"dp":13},{"data":{"data":[0,0,0,30],"type":"Buffer"},"datatype":2,"dp":17},{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":60166}' from endpoint 1 with groupID null
Zigbee2MQTT:info  2022-04-29 22:50:13: MQTT publish: topic 'zigbee2mqtt/0xa4c138bb84e2f10d', payload '{"battery":null,"humidity":null,"last_seen":"2022-04-29T22:50:13+02:00","linkquality":255,"temperature":null}'
Zigbee2MQTT:debug 2022-04-29 22:50:13: Received Zigbee message from '0xa4c138bb84e2f10d', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,229],"type":"Buffer"},"datatype":2,"dp":1},{"data":{"data":[0,0,0,53],"type":"Buffer"},"datatype":2,"dp":2},{"data":{"data":[0,0,0,100],"type":"Buffer"},"datatype":2,"dp":4},{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":60422}' from endpoint 1 with groupID null
Zigbee2MQTT:info  2022-04-29 22:50:13: MQTT publish: topic 'zigbee2mqtt/0xa4c138bb84e2f10d', payload '{"battery":null,"humidity":null,"last_seen":"2022-04-29T22:50:13+02:00","linkquality":255,"temperature":null}'
Zigbee2MQTT:debug 2022-04-29 22:50:14: Received Zigbee message from '0xa4c138bb84e2f10d', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,230],"type":"Buffer"},"datatype":2,"dp":1},{"data":{"data":[0,0,0,53],"type":"Buffer"},"datatype":2,"dp":2},{"data":{"data":[0,0,0,100],"type":"Buffer"},"datatype":2,"dp":4},{"data":{"data":[0,0,1,104],"type":"Buffer"},"datatype":2,"dp":10},{"data":{"data":[0,0,0,200],"type":"Buffer"},"datatype":2,"dp":11}],"seq":60678}' from endpoint 1 with groupID null
Zigbee2MQTT:info  2022-04-29 22:50:14: MQTT publish: topic 'zigbee2mqtt/0xa4c138bb84e2f10d', payload '{"battery":null,"humidity":null,"last_seen":"2022-04-29T22:50:14+02:00","linkquality":255,"temperature":null}'
Zigbee2MQTT:debug 2022-04-29 22:50:14: Received Zigbee message from '0xa4c138bb84e2f10d', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,70],"type":"Buffer"},"datatype":2,"dp":12},{"data":{"data":[0,0,0,20],"type":"Buffer"},"datatype":2,"dp":13},{"data":{"data":[0,0,0,30],"type":"Buffer"},"datatype":2,"dp":17},{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":60934}' from endpoint 1 with groupID null
Zigbee2MQTT:info  2022-04-29 22:50:14: MQTT publish: topic 'zigbee2mqtt/0xa4c138bb84e2f10d', payload '{"battery":null,"humidity":null,"last_seen":"2022-04-29T22:50:14+02:00","linkquality":255,"temperature":null}'
Zigbee2MQTT:debug 2022-04-29 22:50:16: Received Zigbee message from '0xa4c138bb84e2f10d', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0,0,0,229],"type":"Buffer"},"datatype":2,"dp":1},{"data":{"data":[0,0,0,53],"type":"Buffer"},"datatype":2,"dp":2},{"data":{"data":[0,0,0,100],"type":"Buffer"},"datatype":2,"dp":4},{"data":{"data":[0,0,1,104],"type":"Buffer"},"datatype":2,"dp":10},{"data":{"data":[0,0,0,200],"type":"Buffer"},"datatype":2,"dp":11}],"seq":61190}' from endpoint 1 with groupID null`

Reply:
Can you try applying the changes from https://github.com/Koenkk/zigbee-herdsman-converters/commit/221f72f117833f9521ea133584ef9dbad8e243ce ?

Reply:
I tried it now, it didn't help on it's own, but after adding 'commandDataReport' to the type array it started working :)
I also tried reverting the changes and only adding the type, but it stopped working again.
Now the remaining problem is that it keeps complaining about unknown DP's for 12, 13 and 17.

Reply:
You can either ignore it or find out what these data points mean: https://www.zigbee2mqtt.io/advanced/support-new-devices/03_find_tuya_data_points.html#requirements-and-caveats